build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1471,6 +1471,12 @@ message ExecuteOperationMetadata {
   // [ByteStream.Read][google.bytestream.ByteStream.Read] to stream the
   // standard error from the endpoint hosting streamed responses.
   string stderr_stream_name = 4;
+
+  // Additional implementation-specific metadata to send along with the
+  // Operation. Servers MAY choose to populate this field with a partially
+  // complete [ExecutedActionMetadata][build.bazel.remote.execution.v2.ExecutedActionMetadata]
```

Comment:
In that case, please just embed `ExecutedActionMetadata` into `ExecuteOperationMetadata` directly, without using the `repeated google.protobuf.Any` field:
```proto
ExecutedActionMetadata partial_execution_metadata = 5;
```
The current approach introduces an ambiguity, where there's both an `auxiliary_metadata` field in the `ExecuteOperationMetadata` and  `ExecutedActionMetadata` message. Which one should be used for which purpose?

Reply:
Yeah that's fair enough, and `partial_execution_metadata` is a name that makes it clear enough that the fields may not all be present.

Should we explicitly mark this data as optional with a comment like "The server MAY omit this data"?